tekton/release-cheat-sheet.md

```diff
@@ -6,13 +6,20 @@ the pipelines repo, a terminal window and a text editor.
 
 1. `cd` to root of Pipelines git checkout.
 
-2. Point `kubectl` at dogfooding cluster.
+1. Configure `kubectl` to use the  dogfooding cluster.
 
     ```bash
+    # This will update your kubectl to automatically connect to the dogfood cluster.
+    # Remember to point it back to your own cluster afterward, or configure acesss
```

Comment:
I'd argue this comment isn't necessary and actually hinders the "cheat-sheet"-ness here.  The line before this comment says "Configure `kubectl` to use the dogfooding cluster" which is pretty much the same thing as "This will update your kubectl to connect to the dogfood cluster".  And then the comment about changing back to your own cluster is a dedicated listed action at the end of this cheat-sheet which tells the user to "Stop pointing `kubectl` at dogfooding cluster." with the kubectl command to use.

Generally speaking in a "cheat-sheet" this kind of commentary tends to slow me down rather than help me, and in addition it's asking me to hold something in my working memory while I go through a bunch of other steps, which is a little antithetical to the doc's purpose.

If the action at the end which describes how to switch back to your own cluster isn't enough I suggest updating it to be more helpful rather than add exposition in earlier sections.

Reply:
>  If the action at the end which describes how to switch back to your own cluster isn't enough I suggest updating it to be more helpful rather than add exposition in earlier sections.

Personally I find that my strict adherence to the steps in the cheat sheet wanes the further I get down the list - and maybe more importantly, if something interrupts me halfway through, this would leave me in a dangerous state where hopefully I get to the last item in the cheat sheet list but if I don't, I might modify the CI cluster

> it's asking me to hold something in my working memory while I go through a bunch of other steps

I feel like its worse to ask you to definitely complete all the steps and hold in your working memory that you are by default connected to a "production" cluster 😬 

@sbwsg what if we default to using the `--context` arg in the instructions? maybe we could assume you have a `--context dogfooding` context configured or similar, and link to some docs on how to set that up?

Reply:
> @sbwsg what if we default to using the --context arg in the instructions? maybe we could assume you have a --context dogfooding context configured or similar, and link to some docs on how to set that up?

Seems like a reasonable compromise!  👍

Reply:
okay updated @sbwsg lemme know what you think!